### PR TITLE
Mail sender

### PR DESCRIPTION
**Pull request by OpenSourceSimon**
Mail sender, secured with a password. It's publicly available, but a password is required (that's encrypted). It's pinged by GitHub Actions (don't forget to change the URL when merging to main). Credit to @kappq for making most of the code.

### CODE REVIEW

**Review by LevaniVashadze — APPROVED** [review]
Looks good just seems like some weirdness with git

**Comment by OpenSourceSimon** [comment]
Kappa and I have tested it, ready to merge?

File: newspaper/templates/signup.html [hunk 1]
```diff
@@ -40,12 +40,30 @@
                     </select>
                     <select name="time-selection" required class="time-selection">
                         <option value="" disabled selected hidden>Time</option>
-                        <option value="5">5:00 AM</option>
-                        <option value="6">6:00 AM</option>
-                        <option value="7">7:00 AM</option>
-                        <option value="8">8:00 AM</option>
-                        <option value="9">9:00 AM</option>
+                        <option value="01">1:00 AM</option>
```
Comment (LevaniVashadze) [inline newspaper/templates/signup.html:43]: maybe use the 24 Hour system?

**Review by LevaniVashadze — APPROVED** [review]
Looks good to me just check that one comment

**Pull request merged**